lotto/script.js

```diff
@@ -0,0 +1,147 @@
+const purchaseButton = document.querySelector('#purchaseBtn');
+const genButton = document.querySelector('#genBtn');
+const purchaseCountInput = document.getElementById("purchaseCount");
+const purchaseQuantityInput = document.getElementById('purchaseQuantity');
+const totalPriceSpan = document.getElementById('totalPrice');
+const whole = document.querySelector(".whole");
+
+let correctNumbers = [];  
+let userNumbersList = [];      
+
+function genLotto(){
+    let numbers = new Set();
+
+    while (numbers.size<6){
+        function genRandomNumbers(min, max){
+            min = Math.ceil(min);
+            max = Math.floor(max);
+            return Math.floor(Math.random()*(max-min+1))+min;
+        }
+
+        let randomNumber = genRandomNumbers(1,45);
+        numbers.add(randomNumber)
+        numbers = new Set([...numbers].sort((a, b) => a - b));
+    }
+
+    return [...numbers];
+}
+
+function genLottoes(lottoCount) {
+    let lottoList = [];
+    for (let i = 0; i < lottoCount; i++) {
+        lottoList.push(genLotto());
+    }
+    return lottoList;
+}
+
+purchaseQuantityInput.addEventListener('input', function () {
+  const count = parseInt(purchaseQuantityInput.value) || 0;
```

Comment:
```suggestion
purchaseQuantityInput.addEventListener('input', function (event) {
  const count = parseInt(event.target.value);
```
이런식으로 event 객체를 사용해서 값을 표현할 수 있습니다. 실제로 자주 사용하는 방식이기도 하고, 이벤트가 발생했을 때 값을 가져오기 때문에 더 의도에 맞게 표현할 수 있습니다.